H2O-3/introduction-to-machine-learning-with-h2o-3-automl/introduction-to-machine-learning-with-h2o-3-automl.md

```diff
@@ -1,576 +1,938 @@
+
+Now, let’s make some predictions on our test set.
+
+~~~python
+pred = gbm.predict(test)
+pred = pred.cbind(test['ORIGINAL_INTEREST_RATE'])
+pred.head()
+~~~
+
+![automl-preds-reg](assets/automl-preds-reg.png)
+
+Please note that we just combined the response column from our test frame to our predictions to see how the predictions compare to the actual value. 
+
+Out of the first ten predictions, most of them are very close to the actual values, with the exception of some predictions, such as the seventh prediction, which is **6.89,** compared to the actual value which is **8.75.** Because the RMSE is higher than the MAE, we can deduce that we have a couple of instances similar to the one mentioned above, because the larger errors are penalized more. 
+
+Now, let’s try to do what we just did one more time, but this time in Flow.
+
+**Task 5 - R Version**
+
+For our regression use-case, we are using the same dataset and the same training and test sets. But we do need to choose our predictors and response columns, and we will do it as follow:
+
+~~~r
+# Identify predictors and response for the regression use case
+ignore_reg <- c("ORIGINAL_INTEREST_RATE", "FIRST_PAYMENT_DATE", "MATURITY_DATE", "MORTGAGE_INSURANCE_PERCENTAGE",
+            "PREPAYMENT_PENALTY_MORTGAGE_FLAG", "LOAN_SEQUENCE_NUMBER", "PREPAID",
+            "DELINQUENT", "PRODUCT_TYPE")
+
+y_reg <- "ORIGINAL_INTEREST_RATE"
+
+x_reg <- setdiff(colnames(train), ignore_reg)
+x_reg
+~~~
+
+Now we are ready to start our second AutoML model and train it. This time we will only use a time constrain, and set `max_runtime_secs` to `900 seconds`, or fifteen minutes. You will notice that we are specifying the stopping metric and also the sort metric. In the second tutorial, we focused on RMSE and MAE to check the performance of our model, and we noticed that the two values seemed very correlated. For that reason, we could use any of those metrics. For this task, we will use the RMSE for early stopping because it penalizes the error more than the MAE, and we will also use it to sort the leaderboard based on the best value.
+
+~~~r
+#Run AutoML for 15 minutes
+aml_reg <- h2o.automl(x = x_reg,
+                      y = y_reg,
+                      training_frame = train,
+                      max_runtime_secs = 900,
+                      stopping_metric = 'RMSE',
+                      seed = 42,
+                      project_name = "regression",
+                      sort_metric = 'RMSE')
+~~~
+
+Once it’s done, print the leaderboard.
+
+~~~r
+# View the AutoML Leaderboard
+lb <- h2o.get_leaderboard(aml_reg)
+h2o.head(lb, n = -1)
+~~~
+
+![r-automl-reg-leaderboard_1](assets/r-automl-reg-leaderboard_1.png)
+
+**Note:** In the table above, only the top ten models are shown. You can view the other models by clicking **Next** button in your table. 
```

Comment:
in the following sentence remove the word button: "**Note:** In the table above, only the top ten models are shown. You can view the other models by clicking **Next** button in your table. "